data/pages-content/sponsor/information.md

```diff
@@ -10,6 +10,9 @@ subtitle:
 
 
 ### Due Dates of Deliverables
+<h5> Variable Dates</h5>
```

Comment:
To be honest, I had to google what does this phrase mean 🙃
Maybe we could use something like "Flexible Dates" instead?

Reply:
I don't want to give the impression that the dates are flexible. So I now just put it as "go read for details".

Reply:
sounds good